system-test/testnet-performance/upload_results_to_slack.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+upload_results() {
+  payLoad="$(cat <<EOF
+{
+"blocks": [
+ 		{
+			"type": "section",
+			"text": {
+				"type": "mrkdwn",
+				"text":
+"$BUILDKITE_MESSAGE
+Commit SHA: $BUILDKITE_COMMIT
+Link to Build: $BUILDKITE_BUILD_URL
+"
+			},
+		},
+		{
+			"type": "divider"
+		},
+ 		{
+			"type": "section",
+			"text": {
+				"type": "mrkdwn",
+				"text": "Result Details:
+$RESULT_DETAILS"
+			},
+		}
+	]
+}
+EOF
+)"
+
+curl -X POST \
+-H 'Content-type: application/json' \
+--data "$payLoad" \
+"$SLACK_WEBHOOK_URL"
```

Comment:
Looks like SLACK_WEBHOOK_URL and all the other global variables that this function assumes are not checked for, which will make debugging hard